src/rpc_client/src/connector_client.rs

```diff
@@ -101,6 +113,65 @@ impl ConnectorClient {
             })?
             .into_inner())
     }
+
+    pub async fn start_sink_stream(
+        &self,
+        sink_type: String,
+        properties: HashMap<String, String>,
+        table_schema: Option<TableSchema>,
+    ) -> Result<(UnboundedSender<SinkStreamRequest>, Streaming<SinkResponse>)> {
+        let (request_sender, request_receiver) = unbounded_channel::<SinkStreamRequest>();
+
+        // Send initial request in case of the blocking receive call from creating streaming request
+        request_sender
+            .send(SinkStreamRequest {
+                request: Some(SinkRequest::Start(StartSink {
+                    sink_config: Some(SinkConfig {
+                        sink_type,
+                        properties,
+                        table_schema,
+                    }),
+                })),
+            })
+            .map_err(|err| RpcError::Internal(anyhow!(err.to_string())))?;
+
+        let response = self
+            .0
+            .to_owned()
+            .sink_stream(Request::new(UnboundedReceiverStream::new(request_receiver)))
+            .await
+            .map_err(RpcError::GrpcStatus)?
+            .into_inner();
+
+        Ok((request_sender, response))
+    }
+
+    pub async fn validate_sink_properties(
+        &self,
+        connector_type: String,
+        properties: HashMap<String, String>,
+        table_schema: Option<TableSchema>,
+    ) -> Result<()> {
+        let response = self
+            .0
+            .to_owned()
+            .validate_sink(ValidateSinkRequest {
+                sink_config: Some(SinkConfig {
+                    sink_type: connector_type,
+                    properties,
+                    table_schema,
+                }),
+            })
+            .await
+            .inspect_err(|err| {
+                tracing::error!("failed to validate sink properties: {}", err.message())
+            })?
+            .into_inner();
+        response.error.map_or_else(
```

Comment:
```
Ok(response.error.map_err(|err| Err(RpcError::Internal(anyhow!(err.error_message)))?)
```

Reply:
Better to add some more meaningful message in the error, such as `Sink cannot pass validation: {:?}`.